Microsoft.NET.Build.Containers/Resources/Strings.Designer.cs

```diff
@@ -223,7 +223,7 @@ internal class Strings {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Unrecognized mediaType &apos;{0}&apos;..
+        ///   Looks up a localized string similar to CONTAINER2001: Unrecognized mediaType &apos;{0}&apos;..
```

Comment:
Not directly related to this PR but why are we using the old designer generation here, rather than the MSBuild-integrated one that works everywhere and doesn't require this stuff to be checked in?

Reply:
Do you mean GenerateResource task? Good question. Does it generate also properties matching resource name? I have to take a look.

Reply:
Yes, it can (but it's a bit of a pain to turn it all on today: dotnet/msbuild#4751).